soc/arm64/arm/fvp_aemv8a/mmu_regions.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2021 Carlo Caione <ccaione@baylibre.com>
+ * Copyright (c) 2022 Arm Limited (or its affiliates). All rights reserved.
```

Comment:
Please, refrain from adding a copyright change when the change is not big or consistent.

Reply:
ok I will remove that and repush